pages/wizard/product_selection.py

```diff
@@ -44,15 +49,41 @@ def click_openshift(self):
         return self.openshift_checkbox.click()
 
     def select_products(self, products):
-        if 'rhv' in products:
+        if 'rhv' in products and not self.rhv_checkbox.is_selected():
             self.click_rhv()
-        if 'osp' in products:
+        if 'osp' in products and not self.openstack_checkbox.is_selected():
             self.click_openstack()
-        if 'cfme' in products:
+        if 'cfme' in products and not self.cloudforms_checkbox.is_selected():
             self.click_cloudforms()
-        if 'ocp' in products:
+        if 'ocp' in products and not self.openshift_checkbox.is_selected():
             self.click_openshift()
 
+    def requirement_block_parser(self, product):
+        # Requires all the products selected.
```

Comment:
Purely for simplicity. I could do much sophisticated function but right now I don't see a reason why.